code/__HELPERS/_lists.dm

```diff
@@ -134,6 +136,7 @@
 	} while(FALSE)
 
 #define SORT_FIRST_INDEX(list) (list[1])
+#define SORT_COMPARE_DIRECTLY(thing) (thing)
```

Comment:
Unused?

Reply:
Nope https://github.com/tgstation/tgstation/pull/66709/files#diff-e53779f3667517f78b4cea58163350fc8cd21868ee0977e27d580690c35fd456R525

Reply:
The diff you linked me does not appear to point to anything

Reply:
![image](https://user-images.githubusercontent.com/58055496/167090268-91596839-7461-444f-980d-f49fdd71b142.png)
Fucking github